src/nav.component.tsx

```diff
@@ -9,8 +9,9 @@ function getPatientUuidFromUrl() {
 
 export default function PatientChartNavMenu() {
   const patientUuid = getPatientUuidFromUrl();
-  const basePath = spaBasePath.replace(":patientUuid", patientUuid);
-
+  const basePath = spaBasePath
```

Comment:
We use the basePath on a couple of occassions (as input for extension slots). Can we make this consistent / reusable?

Why is the regex so complicated? Shouldn't just a trailing slash be removed? Wouldn't `.replace(/\/$/, "");` be sufficient?

Reply:
I have just tried out the regex above but it's not working.. It still returns such a URL:

```shell
https://openmrs-spa.org/openmrs/spa//patient/54c77c2c-a53f-41ca-a167-705ae0141b9f/chart/immunizations
```

Reply:
What's wrong with this URL? This URL has no trailing slash, which is what the regex should do.

Reply:
It's the extra trailing slash after `spa`. But yeah, I don't see why removing the trailing slash from the base path shouldn't work.